src/gui/extras/19-base_uri_envsubst_entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+PUBLIC_BASE_PATH=${TARANIS_BASE_PATH:-/}
+
+if [ "$PUBLIC_BASE_PATH" != "/" ]; then
+    PUBLIC_BASE_PATH=$(echo "$PUBLIC_BASE_PATH" | sed -e 's/^\/*//' -e 's/\/*$//')
```

Comment:
Is this robust enough? This is just an honest question. Why is it preferred before bash string manipulation using Parameter expansion?

Reply:
It was a bit more complicated than it needed to be, I simplified it